packages/cli/src/commands/deploy.ts

```diff
@@ -21,6 +21,7 @@ import { logger } from '@salto-io/logging'
 import { Workspace } from '@salto-io/workspace'
 import { mkdirp, writeFile } from '@salto-io/file'
 import path from 'path'
+import { DetailedChangeId, DeploySummaryResult } from '@salto-io/core/src/core/deploy'
```

Comment:
```suggestion
import { DetailedChangeId, DeploySummaryResult } from '@salto-io/core'
```
1. import from packages should be at the top (above path)
2. when we import from package we dont write the exact path - see the others imports above. so we need to make sure that core is actually export this types